app/consumers/inventory_events_consumer.rb

```diff
@@ -12,13 +12,6 @@ def process(message)
 
     handle_report_parsing
 
-    case @msg_value['type']
-    when 'delete'
-      DeleteHost.perform_async(@msg_value)
```

Comment:
Even-though it no longer would have our custom Host records, it would still have host assignments. We should find a way how to ensure that host assignments are removed/cleaned after a host is removed.

Reply:
I've added the DeleteHost job back for now. I'm not sure of a better way to solve this other than running a periodic DB cleanup job that looks for dangling references. IMO we kind of need to add such a job regardless, since it's possible that we could miss kafka messages.